source/analysis/dynamic_analysis.py

```diff
@@ -3,6 +3,9 @@
 from os.path import join as os_join
 
 from source.analysis.analysis_type import AnalysisType
+from source.solving_strategies.strategies.linear_solver import LinearSolver
```

Comment:
Should be only imported where the solver is created. It shoould aready be done there.